.github/CODEOWNERS

```diff
@@ -50,62 +50,51 @@
-api-report/map.api.md                                                     @microsoft/fluid-cr-dds
-api-report/matrix.api.md                                                  @microsoft/fluid-cr-dds
-api-report/merge-tree.api.md                                              @microsoft/fluid-cr-dds
-api-report/odsp-driver-definitions.api.md                                 @microsoft/fluid-cr-odsp
-api-report/odsp-driver.api.md                                             @microsoft/fluid-cr-odsp
-api-report/ordered-collection.api.md                                      @microsoft/fluid-cr-dds
-api-report/pact-map.api.md                                                @microsoft/fluid-cr-dds
-api-report/register-collection.api.md                                     @microsoft/fluid-cr-dds
-api-report/replay-driver.api.md                                           @microsoft/fluid-cr-drivers
-api-report/request-handler.api.md                                         @microsoft/fluid-cr-runtime
-api-report/routerlicious-driver.api.md                                    @microsoft/fluid-cr-server
-api-report/runtime-definitions.api.md                                     @microsoft/fluid-cr-runtime
-api-report/runtime-utils.api.md                                           @microsoft/fluid-cr-runtime
-api-report/sequence.api.md                                                @microsoft/fluid-cr-dds
-api-report/shared-object-base.api.md                                      @microsoft/fluid-cr-framework
-api-report/shared-summary-block.api.md                                    @microsoft/fluid-cr-runtime
-api-report/synthesize.api.md                                              @microsoft/fluid-cr-runtime
-api-report/task-manager.api.md                                            @microsoft/fluid-cr-dds
-api-report/telemetry-utils.api.md                                         @microsoft/fluid-cr-telemetry
-api-report/test-driver-definitions.api.md                                 @microsoft/fluid-cr-drivers
-api-report/test-runtime-utils.api.md                                      @microsoft/fluid-cr-runtime
-api-report/test-utils.api.md                                              @microsoft/fluid-cr-test
-api-report/tool-utils.api.md                                              @microsoft/fluid-cr-tools
-api-report/tree2.api.md                                                   @microsoft/fluid-cr-tree
-api-report/undo-redo.api.md                                               @microsoft/fluid-cr-framework
-api-report/view-adapters.api.md                                           @microsoft/fluid-cr-framework
-api-report/view-interfaces.api.md                                         @microsoft/fluid-cr-framework
+# The packages listed here are not necessarily considered part of the public API.
+packages/framework/**/api-report/                      @microsoft/fluid-cr-framework
```

Comment:
Simplified the entries here to bucket by paths. We no longer need to point to individual files to accomplish what we want.

Note that I attempted to preserve the exact behavior, so additional matches were required beyond `packages/framework`, `packages/dds`, etc.